legend-engine-ide-lsp-server/src/main/java/org/finos/legend/engine/ide/lsp/server/LegendLanguageService.java

```diff
@@ -546,4 +546,21 @@ public CompletableFuture<LegendSDLCFeature.SubtypeInfoResult> getSubtypeInfo()
             }
         });
     }
+
+    public CompletableFuture<String> getCurrentUserId()
```

Comment:
We should be able to do this in the client side.

Reply:
Makes sense. I've removed this code.